nautobot/extras/tests/test_views.py

```diff
@@ -1178,6 +1178,19 @@ def setUp(self):
             "choices-MAX_NUM_FORMS": "1000",
         }
 
+    def get_deletable_object(self):
```

Comment:
Probably need the same for `test_api.py`?

Reply:
Yeah, we do have those functions for `test_api.py`